ubuntu-deploy/index.md

```diff
@@ -0,0 +1,414 @@
+
+4. **Basic Command-Line Skills:** Basic familiarity with Linux command-line operations, including file uploads using `scp` and editing files with `vi`.
+
+## Upload Your Local Directus Application Folder to the Server
+
+Assuming you have a local Directus folder structure that looks like this:
+
+```plaintext
+├── database
+│   └── data.db
+├── docker-compose.yml
+├── uploads
+└── extensions
+```
+
+And the _docker-compose.yml_ file looks somewhat like this:
+
+```yaml
+version: '3'
+services:
+  directus:
+    image: directus/directus:latest
+    ports:
+      - 8055:8055
+    volumes:
+      - ./database:/directus/database
+      - ./uploads:/directus/uploads
+      - ./extensions:/directus/extensions
+    environment:
+      KEY: 'replace-with-random-value'
+      SECRET: 'replace-with-random-value'
+      ADMIN_EMAIL: 'admin@example.com'
+      ADMIN_PASSWORD: 'd1r3ctu5'
+      DB_CLIENT: 'sqlite3'
+      DB_FILENAME: '/directus/database/data.db'
+      WEBSOCKETS_ENABLED: 1
+      # Add your other settings
+```
+
+:::info Security Note
+
+Ensure you adjust the email and set a strong password for the admin user.
+
+:::
+
+Use `scp` (Secure Copy Protocol) to upload the local folder to your server.
+
+From your local machine terminal, run:
+
+```bash
+scp -r /path/to/your/local/directus/folder username@server_ip:/path/to/your/remote/folder
+```
+
+Replace _/path/to/your/local/directus/folder_ with the actual local path of your Directus application directory, and _/path/to/your/remote/folder_ with the respective path on your server.
+
+Also, replace `username` with your server's username and `server_ip` with your server's public IP address.
+
+In the example below, I am copying the whole Directus folder to the home directory(`~`) of my server.
+
+![Copying files to the server][image-6]
+
+:::info Database Note
+
+Note that the database used in this tutorial is SQLite. For other types of databases like MySQL and PostgreSQL, you might have to create a database dump and export the dump to your remote server.
+
+:::
+
+## Preparing Your Ubuntu Server
+
+Access your Ubuntu server from your local machine terminal via SSH:
+
+```bash
+ssh username@server_ip
+```
+
+In your server terminal, run the following commands:
+
+```bash
+sudo apt update
+sudo apt upgrade -y
+```
+
+This command updates and upgrades all existing packages to their latest versions.
+
+## Installing Docker and Docker Compose
+
+To install Docker, run the following commands:
+
+```bash
+sudo apt install docker.io -y
+sudo systemctl start docker
+sudo systemctl enable docker
+```
+
+This installs Docker and enables the Docker service.
+To check if Docker is running, run:
+
+```bash
+sudo systemctl status docker
+```
+
+To install Docker Compose, run the following commands:
+
+```bash
+sudo apt install docker-compose -y
+```
+
+To verify that the installation was successful, you can run:
+
+```bash
+docker-compose --version
+```
```

Comment:
This is out of date according to this https://docs.docker.com/engine/install/ubuntu/ `docker.io` is an unofficial package. Can we update the instructions incl the installation of docker compose?